src/main/java/com/snowflake/kafka/connector/internal/streaming/schemaevolution/iceberg/IcebergColumnTypeMapper.java

```diff
@@ -19,23 +18,25 @@
 import org.apache.kafka.connect.data.Time;
 import org.apache.kafka.connect.data.Timestamp;
 
-public class IcebergColumnTypeMapper extends ColumnTypeMapper {
+class IcebergColumnTypeMapper {
+
+  static IcebergColumnTypeMapper INSTANCE = new IcebergColumnTypeMapper();
 
   /**
    * See <a href="https://docs.snowflake.com/en/user-guide/tables-iceberg-data-types">Data types for
    * Apache Iceberg™ tables</a>
    */
-  public static String mapToSnowflakeDataType(Type apacheIcebergType) {
+  String mapToSnowflakeDataType(Type apacheIcebergType) {
     switch (apacheIcebergType.typeId()) {
       case BOOLEAN:
         return "BOOLEAN";
       case INTEGER:
         return "NUMBER(10,0)";
       case LONG:
-        return "NUMBER(19,0)";
+        return "LONG";
```

Comment:
Did it fail with the previous types?

Reply:
- At first run it was resolved fromRecord as "LONG" - add column ... LONG
- After second run it was resolved from icebergSchema as NUMBER(19,0)
- then it failed when alter set data type NUMBER(19,0) was invoked. 

So I made it consise with int64 cause it returns long.

Reply:
* It is a case for sturctured types